opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/NonDominatedSorting.java

```diff
@@ -0,0 +1,179 @@
+package org.opt4j.optimizers.ea;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.opt4j.core.Individual;
+import org.opt4j.core.Objective;
+import org.opt4j.core.Objectives;
+
+/**
+ * Class used for the non-dominated sorting. During non-dominated sorting, the
```

Comment:
Style: do not use "Class" but "The NonDominatedSorting sorts individuals into fronts." or similar.